plugins/frontend/backstage-plugin-github-pull-requests/src/components/PullRequestsTable/PullRequestsTable.tsx

```diff
@@ -137,6 +137,17 @@ export const PullRequestsTableView: FC<Props> = ({
   StateFilterComponent,
   SearchComponent,
 }) => {
+  useEffect(() => {
+    const elem = document.querySelector('.gh-pull-requests-table-header');
+
+    const targetElement = elem?.parentElement?.parentElement?.parentElement;
+
+    if (targetElement) {
+      targetElement.style.display = 'block';
+      targetElement.style.paddingRight = '24px';
+    }
+  }, []);
+
```

Comment:
I do not really like this solution. Can you please try to do this with CSS only? I think something like setting the overflow on the title element would bring us 90% there.

My other thought is that this <Table/> component is coming from backstage/backstage. I think we should try to fix this title in the base component's handling of the title prop so every backstage table would be able to handle the overflow of big titles

Reply:
@kissmikijr Thanks for your comment. I've updated PR to make is CSS only. Result looks almost the same.